lnwallet/btcwallet/btcwallet.go

```diff
@@ -727,6 +727,109 @@ func (b *BtcWallet) RequiredReserve(
 	return reserved
 }
 
+// This is a part of the WalletController interface.
+func (b *BtcWallet) ListAddresses(name string, showCustomAccounts bool) (
```

Comment:
For improved readability/scannability of the code, a function header shouldn't end a line with `(` or `{` if it continues on another line. Otherwise a quick glance might throw the reader off. That's why we would break before `showCustomAccounts` here, to indicate a continuing method declaration with a dangling comma.

To shorten this and the interface declaration, why don't we add a new type for `map[*waddrmgr.AccountProperties][]lnwallet.AddressProperty` in the `interface.go`? It could be called `AccountAddressMap` or something like that?

Reply:
Yes, `AccountAddressMap` would be helpful!